### PR TITLE
[ready] run paket install

### PR DESCRIPTION
**Pull request by 0x53A**
It is slightly frightening that a simple reinstall does these changes, but whatever

image

### CODE REVIEW

File: tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj [hunk 1]
```diff
@@ -265,7 +300,7 @@
     <When Condition="($(TargetFrameworkIdentifier) == '.NETStandard' And ($(TargetFrameworkVersion) == 'v1.3' Or $(TargetFrameworkVersion) == 'v1.4' Or $(TargetFrameworkVersion) == 'v1.5' Or $(TargetFrameworkVersion) == 'v1.6' Or $(TargetFrameworkVersion) == 'v2.0')) Or ($(TargetFrameworkIdentifier) == '.NETCoreApp' And ($(TargetFrameworkVersion) == 'v1.0' Or $(TargetFrameworkVersion) == 'v1.1' Or $(TargetFrameworkVersion) == 'v2.0'))">
       <ItemGroup>
         <Reference Include="System.Collections.NonGeneric">
-          <HintPath>..\..\packages\test\System.Collections.NonGeneric\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
+          <HintPath>..\..\packages\test\System.Collections.NonGeneric.4.3.0\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
```
Comment (matthid) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: did we enable version in path?

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: I changed __nothing__, just executed paket install ...

File: tests/Paket.Tests/ProjectFile/TestData/paket.dependencies [hunk 2]
```diff
@@ -0,0 +1 @@
+#empty file to stop paket.install from messing with files here
```
Comment (matthid) [inline tests/Paket.Tests/ProjectFile/TestData/paket.dependencies]: Good idea. I always reverted/not commited the test folder tbh...

We should make a list what we like to do for breaking changes, just so we can take a look at this list for every major.
Changing this behavior is definitely one of them...

File: tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj (hunk 1 above)
Comment (matthid) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: Yeah just thinking out loud :) This looks suspicious

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: I guess it was inherited from xunit: 
image

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: or not?
image

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: This may be worth looking at ... :\

Comment (matthid) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: Sorry ...

Comment (matthid) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: To elaborate: I don't think it makes sense blocking this, PR but we should open an issue to track that one down then.

Thanks for sending this standalone!

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: I was missing the group with paket why.

It DOES come from NUnit:

image

Comment (0x53A) [inline tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj]: @matthid Yeah, I did a paket install in another branch, cursed at all the changes, reverted everything and sent this =)

**Comment by 0x53A** [comment]
version in path does seem to come from NUnit, but please wait at least for CI anway.

**Comment by 0x53A** [comment]
@matthid do you know if it is possible to update travis to use nightly mono? It would probably fail anyway, but maybe we can then at least write bugreports against mono.

**Comment by matthid** [comment]
Wow you are too fast 👍 .
If I would be cynical I'd say don't waste your time with mono. But I was there before, made everything work (Razor/FSF/...) and with every version the stuff is falling apart again :/ (Sorry it's my frustration speaking).

Yes we can use the nightly channel but I stopped reporting anything some time ago... I just hope for netcore now.

**Comment by 0x53A** [comment]
I don't plan to invest any actual time into this, will probably just tweet every failure at miguel and let him deal with it :D

**Comment by 0x53A** [comment]
k, appveyor is green

**Comment by 0x53A** [comment]
superseded by https://github.com/fsprojects/Paket/pull/2585

**Pull request closed without merging**